### PR TITLE
SIWA: replace Apple image(s) with Apple symbol 

### LINKED ISSUE #392: SIWA: Apple logo background renders weird in press down state
The Apple logo in the "Continue with Apple" logo looks weird during the press-down state of the button. I have dark mode enabled in the simulator when recording this (with animations slowed):

### PR DESCRIPTION
**Pull request by jaclync**
Fixes #392 

Since WCiOS uses the same colors in dark and light mode, the Apple icon has a background color in dark mode. @Garance91540 suggested that we use the Apple symbol instead of static images in dark/light mode, I tried it out and it looks pretty good to me! Please feel free to share any concerns/suggestions/questions that you might have regarding the PR changes, code or design.

## Changes

- In the SIWA CTA, replaced Apple image and a space with `"\u{00a0}\u{00a0}"` so that the Apple symbol naturally matches the button title color

## Testing

- Please check out the testing branch: [`authenticator-testing/apple-icon-symbol` for WPiOS](https://github.com/wordpress-mobile/WordPress-iOS/compare/authenticator-testing/apple-icon-symbol?expand=1) / 
[`issue/authenticator-392-apple-icon` for WCiOS](https://github.com/woocommerce/woocommerce-ios/compare/issue/authenticator-392-apple-icon?expand=1)
- Log out from the app
- Tap on the "Log In" CTA --> the SIWA CTA should look good in both dark and light mode

## Example screenshots

You can notice the SIWA button is slightly shorter after the change and matches the other sign in CTAs more.

### WPiOS

before | after
-- | --
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 12 27 08 | Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 21 05
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 12 27 20 | Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 21 20
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 37 01 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-04 at 10 35 18
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 37 08 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-04 at 10 35 29

### WCiOS

before | after
-- | --
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 11 52 36 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-04 at 10 39 31
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 11 52 42 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-04 at 10 39 41
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 40 59 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-04 at 10 41 10
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 41 07 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-04 at 10 41 16

### CODE REVIEW

**Comment by Garance91540** [comment]
This is looking great, thank you @jaclync !

File: WordPressAuthenticator/Extensions/WPStyleGuide+Login.swift [hunk 1]
```diff
@@ -196,23 +195,9 @@ extension WPStyleGuide {
...
-                                        width: imageSize.width, height: imageSize.height)
-
-        let buttonString = NSMutableAttributedString(attachment: appleAttachment)
-        // Add leading non-breaking space to separate the button text from the Apple logo.
-        let appleTitle = "\u{00a0}" + NSLocalizedString("Continue with Apple", comment: "Button title. Tapping begins log in using Apple.")
-        buttonString.append(NSAttributedString(string: appleTitle))
-
-        return buttonString
+        // Add leading non-breaking space to separate the button text from the Apple symbol.
+        let appleTitle = "\u{00a0}\u{00a0}" + NSLocalizedString("Continue with Apple", comment: "Button title. Tapping begins log in using Apple.")
```
Comment (astralbodies) [inline WordPressAuthenticator/Extensions/WPStyleGuide+Login.swift]: Does this work for a right-to-left language?

**Review by mindgraffiti — COMMENTED** [review]
AFAIK none of the images or symbols (google or apple) are supported to change position for RTL languages. 

@Garance91540 can you measure the new icon vs. the old one? It looks about 5-10 pixels smaller now compared to the old symbol.

**Comment by jaclync** [comment]
> Does this work for a right-to-left language?

@astralbodies yes it does, since the title is appended to the logo attachment in the `NSAttributedString` for the Google CTA. Here's a screenshot:

dark | light
-- | --
Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 10 56 50 | Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 10 58 52

**Comment by Garance91540** [comment]
>@Garance91540 can you measure the new icon vs. the old one? It looks about 5-10 pixels smaller now compared to the old symbol.

Actually I never had icons in my designs, we were always using the symbols. So I can't really measure the new symbol. But you are right that it should be bigger! The Apple symbol should be SF Pro Display 23 px, compared to 17px for the rest of the text. cc @jaclync is it possible to update?

**Comment by jaclync** [comment]
@Garance91540 I tried setting the Apple symbol font to 23pt, here's the comparison (the rightmost is the latest). I actually think the previous update looks more similar to the original size? Just the vertical spacing a little bigger in the original version because of the space in the image.

WPiOS

`develop` | take 1 ( has the same font as the title) | take 2 ( is 23pt)
-- | -- | --
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 12 27 08 | Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 21 05 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-03 at 16 52 13
Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 12 27 20 | Simulator Screen Shot - iPhone 11 Pro - 2020-08-28 at 14 21 20 | Simulator Screen Shot - iPhone 11 Pro - 2020-09-03 at 16 51 54

**Comment by Garance91540** [comment]
I think in the newer versions, the Apple logo size seems to be closer to the one of the Google one, so I think we should keep it! Thank you for working on this!

**Review by mindgraffiti — APPROVED** [review]
Thank you @jaclync!

:shipit:

**Comment by jaclync** [comment]
Thanks for the review @Garance91540 and @mindgraffiti! I updated the PR with these two commits https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/421/commits/00443dd535160af212b10578e834b5994c59bfe4 https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/421/commits/6d4bfc7b35af6cc1910592cacc53bd77334b42c1 and also the screenshots in the PR description with the new size.

I'm merging this for code freeze but if there are any suggestions on the new commits I can update in a separate PR!

**Pull request merged**